src/main/java/build/buildfarm/common/config/ExecutionWrappers.java

```diff
@@ -44,42 +44,43 @@ public class ExecutionWrappers {
    * @brief The program to use when running actions under bazel's sandbox.
    * @details This program is expected to be packaged with the worker image.
    */
-  public String linuxSandbox = "/app/build_buildfarm/linux-sandbox";
+  public String linuxSandbox = "/app/build_buildfarm/src/main/execution_wrappers/linux-sandbox";
 
   /**
    * @field asNobody
    * @brief The program to use when running actions as "as-nobody".
    * @details This program is expected to be packaged with the worker image. The linux-sandbox is
    *     also capable of doing what this standalone programs does and may be chosen instead.
    */
-  public String asNobody = "/app/build_buildfarm/as-nobody";
+  public String asNobody = "/app/build_buildfarm/src/main/execution_wrappers/as-nobody";
```

Comment:
I dislike the idea that we're referencing src/main in runtime applications. Is there a way to relocate this as a part of the deployment?

Reply:
Maybe as a class_path resource.  I'll try.